src/ParagraphsPastePluginBase.php

```diff
@@ -95,7 +102,10 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function build($input) {
-    $property_path = explode('.', $this->pluginDefinition['property_path']);
+    if (empty($this->propertyPath)) {
+      throw new \Exception('Property path not set');
```

Comment:
I don't like throwing a random exception. Whats the advantage in comparison to failing at entityTypeManager->getDefinition()?